appshell/cef_main_window.cpp

```diff
@@ -108,10 +144,14 @@ BOOL cef_main_window::Create()
     int top    = CW_USEDEFAULT;
     int width  = CW_USEDEFAULT;
     int height = CW_USEDEFAULT;
+
     int showCmd = SW_SHOW;
 
     LoadWindowRestoreRect(left, top, width, height, showCmd);
 
+    // make sure the window is visible 
+    EnsureWindowRectVisibility(left, top, width, height);
```

Comment:
I tested this with multiple monitors only so please test with a single monitor.

I tested this by changing various registry values to make sure that when the restore coordinates were offscreen that brackets let Windows figure out where to place the window.

Tested Maximized windows to ensure that the restored placement was correct.  I also tested that, if the restored size was off screen, Windows figures out where to place the window when it is restored.

For reference: https://github.com/adobe/brackets-shell/pull/144